Tests/InkTests/HTMLTests.swift

```diff
@@ -93,6 +93,16 @@ final class HTMLTests: XCTestCase {
 
         XCTAssertEqual(html, "<p>Hello</p><br/><p>World</p>")
     }
+  
+    func testHTMLCommentInMarkdown() {
+      let html = MarkdownParser().html(from: """
+      Hello
+      <!-- some comment -->
+      World
+      """)
+
+      XCTAssertEqual(html, "<p>Hello</p><br/><p>World</p>")
```

Comment:
I'd expect this to be:
```
<p>Hello</p><!-- some comment --><p>World</p>
```
I don't think Ink should strip out comments.

Reply:
Additionally, the CommonMark spec doesn’t strip comments.